ReflectionAnalyzers.Tests/ReflectionAnalyzers.Tests.csproj

```diff
@@ -12,6 +12,14 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\ReflectionAnalyzers.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Gu.Roslyn.Asserts" Version="3.1.0" />
+    <PackageReference Include="Gu.Roslyn.Extensions" Version="0.5.1-dev" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
+    <PackageReference Include="NUnit" Version="3.12.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.13.0" />
```

Comment:
📝 This one should use `PrivateAssets="all"`

```suggestion
    <PackageReference Include="NUnit3TestAdapter" Version="3.13.0" PrivateAssets="all" />
```

Reply:
@sharwell Why? The tests project isn't being packed or depended on by anything else.

Reply:
(I haven't heard of a reason to put that on `NUnit3TestAdapter` before, so I'm curious.)